### ISSUE TITLE
[Not ready for Triage/Review Meeting][Win32][Accessibility] [A11yMAS][Keyboard Navigation] [MAS2.4.3][ Focus is moving to noninteractive element before Run button.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#725]
**User Experience:**
If focus moves to noninteractive element elements, keyboard user will be confused and not able to identify where the focus is present.

**Test Environment:**
OS Build: 16299.19
Excel version:1712(8808.1000)
Tools Used: Narrator
**Prerequisites:**
Add script lab “Add ON” In Excel

**Attachments:**
[MAS2.4.3_Focus moves to non interactive.zip](https://github.com/OfficeDev/script-lab/files/1473633/MAS2.4.3_Focus.moves.to.non.interactive.zip)

**Repro Steps:**
1.	Open Script Lab in Excel
2.	Navigate to Run button and activate it.
3.	Navigate to Run pane Using F6 key
4.	Navigate to “Run” button  using tab key.
5.	Press “Shift +tab key” and verify whether focus is moving to noninteractive element or not.

**Current Behavior:**
Focus is moving to noninteractive elements before run button(focus is not visible normally) and Screen reader is announcing it as “Run office 365 snippet” but no action is getting performed after pressing enter on it.

**Expected Behavior:**
Focus should directly go to Run Button and should announce it as “Run office 365 snippet” button instead of going separately. 

**MAS Reference:**
MAS 2.4.3:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.3
